### PR TITLE
feat: Make dashboards and categories searchable, prepare dashboard implementation

### PR DESCRIPTION
**Pull request by danielptv**
[ODDSK-220](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-220), partially [ODDSK-218](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-218).

* Dashboards, categories and subcategories can now be found using the search
* `ChartTableFilter` now uses a funnel icon to emphasize its role as a filter rather than a search

### CODE REVIEW

File: src/messages/de.json [hunk 1]
```diff
@@ -23,6 +23,20 @@
         "allEntriesLabel": "Zeile",
         "contains": "enthält"
     },
+    "DetailsPageControls": {
+        "download": "Herunterladen",
+        "embed": "Einbetten",
+        "fullscreen": "Vollbild"
+    },
+    "EmbedModal": {
+        "dashboardBody": "Das Dashboard kann mithilfe des folgenden Codes eingebettet werden.",
```
Comment (Zerdados) [inline src/messages/de.json:32]: This message should end in ":" instead of "."

File: src/messages/de.json [hunk 2]
```diff
@@ -23,6 +23,20 @@
...
+        "dashboardTitle": "Dashboard einbetten",
+        "embedHeight": "Höhe",
+        "embedWidth": "Breite",
+        "keepSearchParams": "Suchparameter beibehalten",
+        "resourceBody": "Die Visualisierung dieser Ressource kann mithilfe des folgenden Codes eingebettet werden.",
```
Comment (Zerdados) [inline src/messages/de.json:37]: Same as for dashboardBody

File: src/messages/en.json [hunk 3]
```diff
@@ -23,6 +23,20 @@
...
+        "embed": "Embed",
+        "fullscreen": "Fullscreen"
+    },
+    "EmbedModal": {
+        "dashboardBody": "This dashboard can be embedded using the following code:",
+        "dashboardTitle": "Embed dashboard",
+        "embedHeight": "Height",
+        "embedWidth": "Width",
+        "keepSearchParams": "Preserve search parameters",
+        "resourceBody": "The visualization for this resource can be embedded using the following code.",
```
Comment (Zerdados) [inline src/messages/en.json:37]: I think you forgot something 😅

**Review by Zerdados — APPROVED** [review]

File: src/messages/en.json (hunk 3 above)
Comment (danielptv) [reply src/messages/en.json]: Ups, only changed it for the dashboard but not for resource 😅

**Pull request merged**